server/src-lib/Hasura/Server/PGDump.hs

```diff
@@ -48,7 +48,8 @@ execPGDump b ci = do
         ExitSuccess   -> Right $ CS.cs (clean stdOut)
         ExitFailure _ -> Left $ CS.cs stdErr
 
-    opts = Q.pgConnString ci : "--encoding=utf8" : prbOpts b
+    connString = T.unpack $ bsToTxt $ Q.pgConnString $ Q.ciDetails ci
```

Comment:
It makes me slightly sad that this is necessary. Really, the connection URI should be of type `URL`, or at least `Text`, so this bytestring-to-text munging shouldn’t be necessary… but it probably isn’t worth the effort to change it.